.gitmodules

```diff
@@ -10,3 +10,6 @@
 	path = thirdparty/rmm
 	url = https://github.com/rapidsai/rmm.git
 	branch = branch-0.6
+[submodule "thirdparty/jitify"]
+	path = thirdparty/jitify
+	url = https://github.com/NVIDIA/jitify.git
```

Comment:
Should we pin this to a specific branch / tag / commit?

Reply:
Will [this](https://github.com/rapidsai/cudf/pull/892/commits/79d5111a3f4427e21227551e81d49df993c07b83) do? Or do I need to specify it in `.gitmodules`.

Reply:
👍

Reply:
We need to specify it in `.gitmodules` other a `git submodule update --init --recursive --remote` will update it to the latest commit of master.

Reply:
I can't seem to find a way to pin this to a commit/tag in the `.gitmodules` file. I read [this](https://git-scm.com/docs/gitmodules#Documentation/gitmodules.txt-submoduleltnamegtupdate) and `update` seems to be the only option. But details on [update](https://git-scm.com/docs/git-submodule#git-submodule-update--init--remote-N--no-fetch--no-recommend-shallow-f--force--checkout--rebase--merge--referenceltrepositorygt--depthltdepthgt--recursive--jobsltngt--ltpathgt82308203) say that the only way to do this is to use `none`. Is that what I should do?